pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -587,11 +587,13 @@ type PoolSpec struct {
 
 	// The replication settings
 	// +optional
-	Replicated ReplicatedSpec `json:"replicated,omitempty"`
+	// +nullable
+	Replicated *ReplicatedSpec `json:"replicated,omitempty"`
```

Comment:
If it's a pointer I don't think you need the `// +nullable`.

Reply:
Are you sure? If I don't have this in, the schema doesn't generate `nullable: true` which I think is needed for allowing Rook to actually get `null` values for those.